python/kfserving/server.py

```diff
@@ -0,0 +1,141 @@
+from http import HTTPStatus
+
+import tornado.ioloop
+import tornado.web
+import argparse
+import os
+import logging
+import json
+
+DEFAULT_HTTP_PORT = 8080
+DEFAULT_GRPC_PORT = 8081
+DEFAULT_PROTOCOL = "tensorflow/serving/proto/predict.proto"
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--http_port', default=DEFAULT_HTTP_PORT,
+                    help='The HTTP Port listened to by the model server.')
+parser.add_argument('--grpc_port', default=DEFAULT_GRPC_PORT,
+                    help='The GRPC Port listened to by the model server.')
+args, _ = parser.parse_known_args()
+
+KFSERVER_LOGLEVEL = os.environ.get('KFSERVER_LOGLEVEL', 'INFO').upper()
+logging.basicConfig(level=KFSERVER_LOGLEVEL)
+
+
+class KFServer(object):
+    def __init__(self):
+        self.models = {}
+        self.http_port = args.http_port
+        self.grpc_port = args.grpc_port
+        self.protocol = DEFAULT_PROTOCOL
+
+        self.model_name = args.model_name
+        self.model_dir = args.model_dir
+        self.local_model_dir = args.local_model_dir
+
+    def start(self, models=[]):
+        # TODO add a GRPC server
+        for model in models:
+            self.register_model(model)
+
+        self._http_server = tornado.web.Application([
```

Comment:
this is defined outside of` __init__`, perhaps in `__init__` we can add `self._http_server = None` ?